x/mongo/driver/topology/server_test.go

```diff
@@ -1207,12 +1209,41 @@ func TestServer_ProcessError(t *testing.T) {
 func includesClientMetadata(t *testing.T, wm []byte) bool {
 	t.Helper()
 
-	doc, err := drivertest.GetCommandFromMsgWireMessage(wm)
-	assert.NoError(t, err)
+	var ok bool
```

Comment:
This is a direct copy and paste of the code removed in this commit: https://github.com/mongodb/mongo-go-driver/commit/44496173f8f03c24842511790fb99df4badf901b